transformerlab/plugins/eleuther-ai-lm-evaluation-harness-mlx/setup.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Clone the repository
+git clone https://github.com/chimezie/lm-evaluation-harness-mlx || { echo "Git clone failed or repository already exists"; exit 1; }
```

Need to be able to run setup over and over (every time version increments) so I think you have to let this fail or handle the case of the repo being there?

---

@dadmobile Pushed the changes now